src/select/select-list.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+
+import { isEqual } from 'lodash';
+
+import { Option } from '../api';
```

Comment:
Wanna go with absolute path?

Reply:
IDE 😈

Reply:
There is some problem when using an alias for importing this specific file.
Will investigate later what is happening :/

Reply:
Any news regarding the investigation?

Reply:
Not sure really, I couldn't figure out what's going wrong. Maybe It's just something about linking.